cmd/swagger/commands/generate/support.go

```diff
@@ -45,39 +43,26 @@ func (s *Support) Execute(args []string) error {
 		DumpData:      s.DumpData,
 		DefaultScheme: s.DefaultScheme,
 		TemplateDir:   string(s.TemplateDir),
-	}
+	}, nil
+}
 
-	if err := generator.GenerateSupport(s.Name, nil, nil, &opts); err != nil {
-		return err
-	}
+func (s *Support) generate(opts *generator.GenOpts) error {
+	return generator.GenerateSupport(s.Name, nil, nil, opts)
+}
 
-	var basepath, rp, targetAbs string
-	var err error
-	basepath, err = filepath.Abs(".")
-	if err != nil {
-		return err
-	}
-	targetAbs, err = filepath.Abs(opts.Target)
-	if err != nil {
-		return err
-	}
-	rp, err = filepath.Rel(basepath, targetAbs)
-	if err != nil {
-		return err
-	}
+func (s *Support) log(rp string) {
 
 	log.Printf(`Generation completed!
 
-For this generation to compile you need to have some packages in your vendor or GOPATH:
+For this generation to compile you need to have some packages in your GOPATH:
 
-  * github.com/go-openapi/runtime
-  * github.com/asaskevich/govalidator
-  * github.com/tylerb/graceful
-  * github.com/jessevdk/go-flags
-  * golang.org/x/net/context/ctxhttp
+	* github.com/go-openapi/runtime
```

Comment:
please retain the list of packages, depending on which vendorint tool you use you need the exhaustive list

Reply:
This was a copy-paste error :D